### ISSUE TITLE
Fails to build WebWorker script for trainAsync

### ISSUE THREAD

**Issue by aurium** [cazala/synaptic#148]
When i call `trainAsync()`, this code:
```hardcode += "var F =  new Float64Array([" + this.optimized.memory.toString() + "]);\n";```
Builds this:
```var F =  new Float64Array([[object Float64Array]]);```

I believe it should replace `toString()` by `join(',')`. It works for me. Is there any other use-case where `this.optimized.memory` is not an array? Then i can send a pull request solving this issue.

**Comment by cazala** [comment]
Are you using the minified version by any chance?

Sent from my iPhone

On Oct 6, 2016, at 1:15 PM, Aur?lio A. Heckert <notifications@github.com<mailto:notifications@github.com>> wrote:


When i call trainAsync(), this code:
hardcode += "var F = new Float64Array([" + this.optimized.memory.toString() + "]);\n";
Builds this:
var F = new Float64Array([[object Float64Array]]);

I believe it should replace toString() by join(','). It works for me. Is there any other use-case where this.optimized.memory is not an array? Then i can send a pull request solving this issue.

-

**Comment by aurium** [comment]
No, i'm using this file: https://github.com/cazala/synaptic/blob/master/dist/synaptic.js downloaded one hour ago.

**Comment by cazala** [comment]
Yes, it is either an Array or a Float64Array, both have `join()`
